README.md

```diff
@@ -40,7 +38,318 @@ Under construction.
 <img src="https://img.shields.io/github/last-commit/memphisdev/memphis-broker?color=61dfc6&label=last%20commit">
 </p>
 
-**[Memphis{dev}](https://memphis.dev)** is an open-source real-time data processing platform<br>
-that provides end-to-end support for in-app streaming use cases using Memphis distributed message broker.<br>
-Memphis' platform requires zero ops, enables rapid development, extreme cost reduction, <br>
-eliminates coding barriers, and saves a great amount of dev time for data-oriented developers and data engineers.
+**[Memphis](https://memphis.dev)** is a next-generation alternative to traditional message brokers.<br><br>
+A simple, robust, and durable cloud-native message broker wrapped with<br>
+an entire ecosystem that enables cost-effective, fast, and reliable development of modern queue-based use cases.<br><br>
+Memphis enables the building of modern queue-based applications that require<br>
+large volumes of streamed and enriched data, modern protocols, zero ops, rapid development,<br>
+extreme cost reduction, and a significantly lower amount of dev time for data-oriented developers and data engineers.
+
+## Installation
+
+```sh
+$ pip3 install memphis-py
```

Comment:
fix it to the most known package manager of Java and write the command to install memphis